### DISCUSSION TITLE
Questão TI21#9

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#16] [Q&A] [answered]
Como você pode criar uma animação suave usando CSS?

**Comment by erickgusson** [comment]
Você pode criar animações usando a função nativa do css Keyframe

**Reply by Guilhlm** [reply]
voce pode definir um width e height de 300, apos isso crie um hover dessa mesma tag com transition 2secs com 20% a mais de tamnho

**Comment by RenonHG** [comment, marked as answer]
Utilizando o comando transition: 500ms.
Por exemplo, isso fará uma alteração de acordo com seus elementos levando 0,5 segundos do inicio ao fim em sua animação
